### PR TITLE
Add combined variable for space and water heating

### PR DESCRIPTION
**Pull request by robinhasse**
This adds a new variable equal to the sum of space heating and water heating variables. The same exists already in Ariadne. Would be great if @danielhuppmann or @phackstock could take a look. Its quite urgently needed for ECEMF T2.2 work. Thanks in any case :)

### CODE REVIEW

File: definitions/variable/energy/energy-final.yaml [hunk 1]
```diff
@@ -106,6 +106,18 @@
     energy_use: all excl. losses
     sector: Residential and Commercial
     unit: [EJ/yr, GWh]
+- Final Energy|Residential and Commercial|Space and Water Heating:
+    description: Final energy consumed for space and water heating in residential and commercial buildings
+    energy_source: all
+    energy_use: all excl. losses
+    sector: Residential and Commercial
+    unit: [EJ/yr, GWh]
```
Comment (phackstock) [inline definitions/variable/energy/energy-final.yaml:114]: Do you need to allow two different variables?
They also seem to have two different dimensions, Energy/time vs. Energy.

File: definitions/variable/energy/energy-final.yaml [hunk 2]
```diff
@@ -106,6 +106,18 @@
...
+- Final Energy|Residential and Commercial|Space and Water Heating|{Final Energy carrier}:
+    description: Final energy consumed for space and water heating in residential and commercial buildings
+      from {Final Energy carrier}
+    energy_use: all excl. losses
+    sector: Residential and Commercial
+    unit: [EJ/yr, GWh]
```
Comment (phackstock) [inline definitions/variable/energy/energy-final.yaml:120]: Same as above

**Review by phackstock — COMMENTED** [review]
Two small comments in line.

File: definitions/variable/energy/energy-final.yaml (hunk 1 above)
Comment (robinhasse) [reply definitions/variable/energy/energy-final.yaml:114]: I kept both units to be consistent with related existing variables. But I go with `EJ/yr` now as for the time being I'll be the only one using it...

**Review by phackstock — APPROVED** [review]
Thanks for the quick update @robinhasse. 
Looks good to me.

**Comment by robinhasse** [comment]
Great, thanks for the quick review @phackstock! I don't have rights to write so one of you would need to merge it eventually.

**Review by danielhuppmann — APPROVED** [review]
Thanks @robinhasse!

**Pull request merged**